Add regex game test plans (#108)
Gamify-IT/docs: dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-1-highscore-not-update.md

# Highscore not update test (`u.regex-game-1`)

Version: V1.0, 07.08.2022 \
Author: Leon Layer \
Tester: -

## Description

> Tests if the high score not increases when a lower or same score is reached.

Reviewer: ```suggestion
Tests if the high score didnt increase when a lower or same score is reached.
```

Reviewer 2: ```suggestion
Tests that the high score doesn't change when score <= highscore.
```

## Precondition

The game is in a state where the maximum possible higscore has not yet been reached.

## Postcondition

> by playing, if the current higscore will not be exceeded, the higscore will be updated in the menu. 

Reviewer: isnt this sentence wrong?

> by playing, if the current higscore will not be exceeded, the higscore will be updated in the menu. 

Reviewer: ```suggestion
by playing, if the current higscore is not exceeded, the higscore is not updated in the menu. 
```

Reviewer 2: ```suggestion
The highscore has its original value.
```

## Typical procedure

1. the tester plays the game but does not have a score of 0.
2. the tester sees his previous higscore in the menu wich is not 0.
3. by trying again, he doesnt surpasses his previous best performance

> 4. the game will not update his higscore in the meu.

Reviewer: ```suggestion
4. the game will not update his highscore in the menu.
```

Reviewer 2: ```suggestion
4. the highscore in the menu is not updated
```

## Alternative procedures

> 2.1. In the menu still 0 or another wrong score is displayed as highscore \
> 3.1. the highscore is updated and a lower highscore is displayed \

Reviewer: ```suggestion
```

as we discussed about alternative
